### DISCUSSION TITLE
What does "Mute" do?

### DISCUSSION THREAD

**Discussion by DNin01** [ScratchAddons/ScratchAddons#4954] [Help] [answered]
What does this option do?
![Right clicking the extension icon shows an option called Mute](https://user-images.githubusercontent.com/106490990/182968721-26aa08d2-399c-4ddd-aa30-2feee454e844.png)

I just need to know for a change I might make (depending on what it does, we might want to change what it says. I was going to do that), and maybe to help clarify it for anyone else. Does it turn off the notifications like a "Do Not Disturb" mode? Does it only stop the notifications from playing sound? I'm sorry, I just wanna be sure.

**Comment by WorldLanguages** [comment, marked as answer]
It hides the message count, and disables notifications from the "Scratch Notifier" feature

**Reply by WorldLanguages** [reply]
BTW, could you tell us what you plan to work on?

**Reply by DNin01** [reply]
Got it. That's what I thought. Didn't know it hid the message count too!

> BTW, could you tell us what you plan to work on?

I was considering changing "Mute" to say something like "Do Not Disturb" or something easier to understand.

**Reply by WorldLanguages** [reply]
> I was considering changing "Mute" to say something like "Do Not Disturb" or something easier to understand.

Do you think it's hard to understand? We use the same terminology as Discord

**Reply by DNin01** [reply]
> > I was considering changing "Mute" to say something like "Do Not Disturb" or something easier to understand.
> 
> Do you think it's hard to understand? We use the same terminology as Discord

It's not too bad, but it might be more self-explanatory if it was changed.

**Reply by mxmou** [reply]
> > I was considering changing "Mute" to say something like "Do Not Disturb" or something easier to understand.
> 
> Do you think it's hard to understand? We use the same terminology as Discord

"Mute" usually refers to sound.

**Reply by Hans5958** [reply]
> We use the same terminology as Discord

"Do Not Disturb" on Discord refers to the status where it won't give you notifications about messages and pings, so this is the more appropriate terminology IMO.

**Reply by WorldLanguages** [reply]
> "Do Not Disturb" on Discord refers to the status where it won't give you notifications about messages and pings, so this is the more appropriate terminology IMO.

Well, "Mute" does the same thing but for a single server, instead of globally. But I think I agree the term "DND" could be used here.
